src/PresentationalComponents/ReportDetails/ReportDetails.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
```

Comment:
ok this whole thing is a federated module that advisor advertises, yah shouldn't keep it here, otherwise that whole drifting of design will happen real fast, lets talk about how yah can consume this from advisor (or checkout https://github.com/RedHatInsights/content-preview which consumes it already or the inventory app)